app/api/tests/test_api.py

```diff
@@ -715,6 +715,40 @@ def test_disallows_non_annotation_approver_to_approve_and_disapprove_labels(self
     def doCleanups(cls):
         remove_all_role_mappings()
 
+class TestCommentListAPI(APITestCase, TestUtilsMixin):
+
+    @classmethod
+    def setUpTestData(cls):
+        cls.project_member_name = 'project_member_name'
+        cls.project_member_pass = 'project_member_pass'
+        create_default_roles()
+        project_member = User.objects.create_user(username=cls.project_member_name,
+                                                  password=cls.project_member_pass)
+        cls.post_data = {'text': 'TEST'}
+        cls.main_project = mommy.make('SequenceLabelingProject', users=[project_member])
+        cls.label1 = mommy.make('Label', project=cls.main_project)
+        cls.label2 = mommy.make('Label', project=cls.main_project)
+        doc1 = mommy.make('Document', project=cls.main_project)
+        mommy.make('Document', project=cls.main_project)
+        mommy.make('SequenceAnnotation', document=doc1, user=project_member, label=cls.label1)
+        cls.url = reverse(viewname='comments_list', args=[cls.main_project.id, doc1.id])
+        #cls.params = {'seq_annotations__label__id': cls.label1.id}
+        assign_user_to_role(project_member=project_member, project=cls.main_project,
+                            role_name=settings.ROLE_ANNOTATOR)
+
+    def test_returns_comments_to_project_member(self):
```

Comment:
Maybe more tests?

Reply:
it would be good to verify what is returned here. create some comments and check for them.

Reply:
there should be a test for showing comments from other users